apsuite/orbcorr/orbit_correction.py

```diff
@@ -0,0 +1,198 @@
+"""."""
+
+from copy import deepcopy as _dcopy
+from collections import namedtuple as _namedtuple
+import numpy as _np
+
+import pyaccel
+
+from .calc_orbcorr_mat import OrbRespmat
+
+
+class CorrParams:
+    """."""
+
+    def __init__(self):
+        """."""
+        self.minsingval = 0.2
+        self.maxkickch = 300e-6  # rad
+        self.maxkickcv = 300e-6  # rad
+        self.maxkickrf = 500e6  # Hz
+        self.maxdeltakickch = 300e-6  # rad
+        self.maxdeltakickcv = 300e-6  # rad
+        self.maxdeltakickrf = 1000  # Hz
+        self.maxnriters = 10
+        self.tolerance = 0.5e-6  # m
+        self.userf = True
+
+
+class OrbitCorr:
+    """."""
+
+    CORR_STATUS = _namedtuple('CorrStatus', ['Fail', 'Sucess'])(0, 1)
+
+    def __init__(self, model, acc):
+        """."""
+        self.acc = acc
+        self.params = CorrParams()
+        self.respm = OrbRespmat(model=model, acc=self.acc, dim='6d')
+        self.respm.model.cavity_on = True
+
+    def get_jacobian_matrix(self):
+        """."""
+        return self.respm.get_respm()
+
+    def get_kicks(self):
+        """Return corrector kicks."""
+        if model is None:
+            model = self.respm.model
+        chidx = self.respm.ch_idx
+        cvidx = self.respm.cv_idx
+        rfidx = self.respm.rf_idx
+        kickch = pyaccel.lattice.get_attribute(model, 'hkick_polynom', chidx)
+        kickcv = pyaccel.lattice.get_attribute(model, 'vkick_polynom', cvidx)
+        kickrf = pyaccel.lattice.get_attribute(model, 'rf_frequency', rfidx)
```

Comment:
The attribute is `frequency` instead of `rf_frequency`

Reply:
fixed!